src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -134,12 +137,27 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
       'dist',
       'codicon.css',
     ]);
-    const jsUri =
-        getUri(webview, this.context.extensionUri, ['media', 'CfgEditor', 'cfgeditor.js']);
+    const jsUri = getUri(webview, this.context.extensionUri, ['media', 'CfgEditor', 'index.js']);
 
     const cssUri =
         getUri(webview, this.context.extensionUri, ['media', 'CfgEditor', 'cfgeditor.css']);
 
+    const htmlPath =
+        vscode.Uri.joinPath(this.context.extensionUri, 'media/CfgEditor/cfgeditor.html');
+    let html = Buffer.from(await vscode.workspace.fs.readFile(htmlPath)).toString();
```

Comment:
Hmm, I think this `Buffer` should be handled with exceptions like file handle and buffer(overflow/underflow). Is it okay not to care about exceptions?

Reply:
I couldn't understand your saying but... as you used overflow and underflow words, are you talking about buffer index error? :sweat_smile:

Reply:
Cases
1. What happen if fs cannot read the file?
2. What happen if fs can read the file and then cannot allocate memory for the file?

Reply:
> What happen if fs cannot read the file?

If `await vscode.workspace.fs.readFile(htmlPath)` returns `null` or `undefined`, `Buffer.from` raises Exception.
But I didn't consider this because we load static file..!

> What happen if fs can read the file and then cannot allocate memory for the file?

I didn't consider this because HTML file is really small...!
And after loading is finished, variable is not used anymore :)

Reply:
@llFreetimell Sounds very reasonable.

(optional) But,
1. I think exception is an exception because we couldn't expect and the way the html contents are loaded could be changed. It would be better to handle exception on this case.
2. There is `Utils/Uri.ts` in our project where helper functions for URI exist. How about make this line a single function and call it for maintainability and readability?

Reply:
> we couldn't expect and the way the html contents are loaded could be changed. It would be better to handle exception on this case.

I agree :)

> How about make this line a single function and call it for maintainability and readability?

I think it is good idea to create separate function with exception handling! That would be definitely good for maintainability and readibility :D
And also I think it would be better for newbies to use `vscode.workspace.fs.readFile` :)
Let's do it in another PR!